_stbt/config.py

```diff
@@ -1,8 +1,14 @@
 import configparser
 import enum
 import os
+import typing
 from contextlib import contextmanager
 
+if typing.TYPE_CHECKING:
```

Comment:
It avoids circular imports and preserves some modules being leaf modules (not importing anything).  Strings don't bother me - at least in VSCode they are highlighted and work with "go to definition", etc just as well as not strings.

We will still need to use lots of strings for forward declarations anyway, so I'm just blindly following this idiom as strings don't hurt, and don't need to be evaluated at import time.  I figure by doing it this way it will have the least impact.

I'm trying to be quite conservative here, particularly given that things may change for Python 3.11 too: https://lwn.net/Articles/858576/ .

On the subject of forward annotations we'll have to discuss our recommendations for customer test-packs WRT page object cross-references and circular imports.